src/common/components/home/FeaturedProfilesSection.tsx

```diff
@@ -228,7 +228,7 @@ const PreviewContainer: FC<any> = (props) => {
   );
 };
 
-const FollowUnfollowButtonDataWrapper: VFC<{ targetPubkey: string; className?: string }> = ({
+export const FollowUnfollowButtonDataWrapper: VFC<{ targetPubkey: string; className?: string }> = ({
```

Comment:
If you're gonna use this outside of `FeaturedProfilesSection`, it should go into `FollowUnfollowButton`

Reply:
Agreed